src/test-helpers/createTestApp.tsx

```diff
@@ -13,10 +13,9 @@ import storage from '../util/storage'
 let cleanup: Await<ReturnType<typeof initialize>>['cleanup']
 
 /** Set up testing and mock document and window functions. */
-const createTestApp = async () => {
+const createTestApp = async ({ tutorial }: { tutorial?: boolean } = {}) => {
```

Comment:
```suggestion
const createTestApp = async ({ tutorial = false }) => {
```

See: https://github.com/cybersemics/em/pull/2702/files#r1919837703